share/spice/missing_kids/missing_kids.js

```diff
@@ -0,0 +1,60 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_missing_kids = function (api_result) {
+        
+        if (!api_result) Spice.failed('missing_kids');
+        if (api_result.error) Spice.failed('missing_kids');
+        
+        var articles = api_result.rss.channel.item;
+        if (articles.length == 0) Spice.failed('missing_kids');
+        if (articles.length > 20) articles = articles.slice(0,20);
+        
+        var script = $('[src*="/js/spice/missing_kids/"]')[0],
```

Comment:
What you have returns the encoded query.  What I'm doing here is getting the return value from handle sub, which is the two-letter state abbreviation.